src/map/modifier.h

```diff
@@ -1000,7 +1001,7 @@ enum class Mod
     // 570 through 825 used by WS DMG mods these are not spares.
     //
     // SPARE IDs:
-    // 141 to 143
+    // 142 to 143
```

Comment:
Depending on when/which PR gets in first, this line may conflict.
I changed the modifiers used in my PR (#5166) to not use yours (141) so that concrete conflict problem wont come up, but just a heads up.
Probably solved by just deleting this line, since I now use mods 142 and 143.